docs/docs/50-user-guide/60-reference-docs/90-events/10-event-reference.md

```diff
@@ -0,0 +1,247 @@
+---
+description: Reference documentation for Kargo events and types.
+---
+
+# Kargo Event Reference
```

Comment:
Probably need the EE/beta markers here?

Reply:
Ah. That's a good point I'd not considered. Do we want to mention then that OSS emits the events, but only EE comes with all the plumbing to do much with them?

Reply:
Yeah I'll probably leave an "info" admonition up top to explain that